docs/source/pypureclient.flasharray.FA_2_0.models.rst

```diff
@@ -412,6 +412,14 @@ pypureclient.flasharray.FA\_2\_0.models.space module
    :undoc-members:
    :show-inheritance:
 
+pypureclient.flasharray.FA\_2\_0.models.space2 module
```

Comment:
how does space2 come about? :)

Reply:
It is a way how swagger-codegen handles classes with the same name, just add number postfix to the name. In this particular case, it's encountered `_space.yaml` first assigned `space` name to the associated class, after that it found `space.yaml` and assigned `space2` name to the associated class.

Reply:
Also it seems like the best approach would be rename `space.yaml` to `space-something.yaml` instead of manipulating `_space.yaml`. Because of the following reasons:
1. `space.yaml` not that widely used as `_space.yaml`
2. Better control over the naming, as this postfixed versions of the classes have limited support, I mean there are more issues with them =)
3. For customers it will look like introducing of the new fields and usual way for these introduction is via new classes

Reply:
Agreed on space-something.yaml. I'm curious how the generated code would look like. Ultimate goal is avoid breaking compatibility.

We might want to consider going a little more extreme, in that sense that when we code-gen, we first merge in the content from `_space.yaml` (or `space-something.yaml`), b/c I feel the inheritance concept with `$ref` is very fragile